### PR TITLE
Non-Ministry User Account creation#9332

### PR DESCRIPTION
**Pull request by shabeeb-aot**
*Issue #:*
https://github.com/bcgov/entity/issues/9332

*Description of changes:*
* setting value to the session on click of check box for GOVN account 
* hiding DROWDWON for GVN account
* submission is pending in this PR

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the sbc-auth license (Apache 2.0).

### CODE REVIEW

File: auth-web/src/components/auth/common/PaymentMethods.vue [hunk 1]
```diff
@@ -203,7 +203,8 @@ export default class PaymentMethods extends Vue {
   }
 
   private get isPADOnly () {
-    return (this.currentOrgType === Account.UNLINKED_PREMIUM)
+    const isGovNAccount = this.currentOrganization?.accessType === AccessType.GOVN
```
Comment (shabeeb-aot) [inline auth-web/src/components/auth/common/PaymentMethods.vue:206]: show only pad if its GOVM account

File: auth-web/src/components/auth/create-account/AccountTypeSelector.vue [hunk 2]
```diff
@@ -198,6 +198,8 @@ export default class AccountTypeSelector extends Mixins(Steppable) {
     }
 
     this.setAccessType(this.getOrgAccessType())
+    const accessType = this.getOrgAccessType()
+    this.setCurrentOrganization({ ...this.currentOrganization, ...{ accessType: accessType } })
```
Comment (shabeeb-aot) [inline auth-web/src/components/auth/create-account/AccountTypeSelector.vue:202]: need to get access type in current org

File: auth-web/src/store/modules/org.ts [hunk 3]
```diff
@@ -923,7 +923,8 @@ export default class OrgModule extends VuexModule {
   @Action({ rawError: true })
   public async resetAccountWhileSwitchingPremium (): Promise<void> {
     this.context.commit('setGrantAccess', false)
-    this.context.commit('setCurrentOrganization', { name: '' })
+    // need to keep accesstype while resetting to know whenther the account is GOVM or not.
+    this.context.commit('setCurrentOrganization', { ...this.context.state['currentOrganization'], ...{ name: '' } })
```
Comment (shabeeb-aot) [inline auth-web/src/store/modules/org.ts:927]: while resetting before we are setting name as null, no we need accesstype in currentorg. If needed we can remove resetting all and add only access type from state.
any thoughts?

File: auth-web/src/views/auth/create-account/non-bcsc/NonBcscAccountSetupView.vue [hunk 4]
```diff
@@ -276,6 +277,8 @@ export default class NonBcscAccountSetupView extends Vue {
         await this.getUserProfile('@me')
         await this.syncOrganization(organization.id)
         await this.syncMembership(organization.id)
+        // remove GOVN accoutn type from session
```
Comment (shabeeb-aot) [inline auth-web/src/views/auth/create-account/non-bcsc/NonBcscAccountSetupView.vue:280]: remove from session once org is saved.

**Review by karthik-aot — APPROVED** [review]

**Review by sumesh-aot — APPROVED** [review]

**Pull request merged**